src/App.js

```diff
@@ -4,11 +4,16 @@ import AddTodoForm from './AddTodoForm';
 
 
 function App() {
+  const [todoList, setTodoList] = React.useState([]);
+    const addTodo = (newTodo) => {
+      setTodoList([...todoList, newTodo])
+  }
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm/>
-      <TodoList/>
+      <AddTodoForm onAddTodo={addTodo}/>
+       
```

Comment:
This is unnecessary empty space. Sometimes it is useful to add blank lines to separate out large chunks of HTML, but here it's not necessary. 

The reason why I'm pointing out these really small problems is that there aren't any big problems with your code! :)